From: =?utf-8?q?Sebastian_Dr=C3=B6ge?= <slomo@ubuntu.com>
Date: Tue, 14 Nov 2006 15:35:00 +0100
Subject: Don't abort on fatal warnings by default

This behaviour can be controlled by the DBUS_FATAL_WARNINGS enviroment
variable. This will be set to upstream default again at some point so
if you have an application that prints a DBus warning get it fixed.

Origin: vendor, Debian
Forwarded: no
---
 dbus/dbus-internals.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dbus/dbus-internals.c b/dbus/dbus-internals.c
index d7ef089..f78613b 100644
--- a/dbus/dbus-internals.c
+++ b/dbus/dbus-internals.c
@@ -185,7 +185,7 @@ const char *_dbus_no_memory_message = "Not enough memory";
 
 static dbus_bool_t warn_initted = FALSE;
 static dbus_bool_t fatal_warnings = FALSE;
-static dbus_bool_t fatal_warnings_on_check_failed = TRUE;
+static dbus_bool_t fatal_warnings_on_check_failed = FALSE;
 
 static void
 init_warnings(void)
